cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -321,15 +321,17 @@ public abstract class AbstractModel {
     /**
      * Constructor
      *
-     * @param reconciliation   The reconciliation
-     * @param resource         Kubernetes resource with metadata containing the namespace and cluster name
-     * @param applicationName  Name of the application that the extending class is deploying
+     * @param reconciliation    The reconciliation marker
+     * @param resource          Custom resource with metadata containing the namespace and cluster name
+     * @param name              Name of the Strimzi component usually consisting from the cluster name and component type
+     * @param componentType     Type of the application that the extending class is deploying
```

Comment:
is "application" referring to the "Strimzi component" above? I know you are referring to Kafka, ZooKeeper, Cruise Control as "applications" but maybe we want to have consistency with "Strimzi component"? So changing in one place or the other?